app/Client/Http/HttpClient.php

```diff
@@ -66,6 +66,7 @@ protected function passRequestThroughModifiers(RequestInterface $request, ?WebSo
     protected function createConnector(): Connector
     {
         return new Connector($this->loop, [
+            'dns' => '127.0.0.1',
```

Comment:
@mpociot this breaks usage with docker, like `laravel sail`... i had to downgrade do `1.4.1` because of this change....

If this is really necessary for `local shared domains` maybe we need to make this configurable or check somehow to set this dynamically

Reply:
Yeah, we had that fixed in #146.
I‘m not that deep into the topic. @mpociot what do you mean by local shared domains? Can‘t they be resolved by the system configured DNS?

Reply:
So the main problem is that Valet, for example, needs to use 127.0.0.1 as a DNS. Otherwise, a local URL such as `my-site.test` would not resolve.

But I agree, for Docker, we do not need the DNS server. So I think it would be best to configure this